libs/services/src/SlippageService/SlippageServiceMain.spec.ts

```diff
@@ -1,47 +1,445 @@
-import { Container, injectable } from 'inversify';
-import { SlippageService, slippageServiceSymbol } from './SlippageService';
 import { SlippageServiceMain } from './SlippageServiceMain';
 import {
   PricePoint,
-  PriceStrategy,
   SupportedChainId,
   UsdRepository,
-  usdRepositorySymbol,
 } from '@cowprotocol/repositories';
+import ms from 'ms';
 
-@injectable()
-class UsdRepositoryMock implements UsdRepository {
-  getUsdPrice(
-    chainId: SupportedChainId,
-    tokenAddress: string
-  ): Promise<number | null> {
-    throw null;
-  }
-  getUsdPrices(
-    chainId: SupportedChainId,
-    tokenAddress: string,
-    priceStrategy: PriceStrategy
-  ): Promise<PricePoint[] | null> {
-    throw null;
-  }
-}
+const MIN_SLIPPAGE = 50;
+const MAX_SLIPPAGE = 200;
```

Comment:
Nitpick: since it matches the implementation, what about just importing the constants from there?